src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -505,11 +519,14 @@ public void completeTaskWithPrivilegeAndCompletionOptions(String taskId,
                 userId,
                 taskStateIsAssignedAlready
             );
+            task.setState(CFTTaskState.COMPLETED);
+            task.setTerminationProcess(terminationProcess);
+            setTaskActionAttributes(task, userId, TaskAction.COMPLETED);
```

Comment:
Not sure if this is from the merge but there are a couple of duplicated lines here?  The state & attributes should already be set on the DB already.